src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
+In this exemple, the [Description] tag is set as step test case in the allure-reporter tool.  
+The [AllureTms] tag contains the test case name in Jama.
+
+Use the following example as a basis for your test cases.
+
+```C#
+    [TestFixture("ACR", "mg/mmol")]
+    [TestFixture("Chol/HDL", "")]
+    [AllureNUnit]
+    public class QCAfinionTestCases : UnitTestBase<AfinionAnalyzer>
+    {
+        string code;
+        string unit;
+        string flag;
+
+        XDocument translatedResult = null;
+        OBSR01 observationMessage = null;
+        HELR01 helloMessage = null;
+
+        public QCAfinionTestCases(string code, string unit)
+        {
+            this.code = code;
+            this.unit = unit;
+        }
+
+        [AllureEpic("TestAfinion")]
```

Comment:
I would like to clarify that the [AllureEpic] is a nice-to-have feature in allure-report and it's not yet developed.